api/v1/zz_generated.deepcopy.go

```diff
@@ -123,6 +123,16 @@ func (in *ExternalServiceSpec) DeepCopyInto(out *ExternalServiceSpec) {
 		*out = new(int32)
 		**out = **in
 	}
+	if in.TargetCPUUtilizationPercentage != nil {
+		in, out := &in.TargetCPUUtilizationPercentage, &out.TargetCPUUtilizationPercentage
+		*out = new(int32)
+		**out = **in
```

Comment:
OOI what does this do?

Reply:
This is autogenerated code allowing deep copying of the object. I guess its a double dereference setting a double pointer?